README.md

```diff
@@ -147,7 +147,7 @@ You need to complete the following procedures in Genesys Cloud and the Genesys C
 **Note:** In the Create a widget for web chat procedure, in the **Route to Flow** menu, choose the flow you created in the earlier procedure.
 
 2. After you save the widget, the saved page displays a Deployment Key string. Copy the Deployment Key string for later use.
-3. From your browser, go to the Genesys Cloud Developer Center and open the Web Chat developer tool. For more information, see [Web Chat developer tool quick start](https://developer.mypurecloud.com/gettingstarted/developer-tools-web-chats.html "Opens the Web Chat developer tool quick start article").
+3. From your browser, go to the Genesys Cloud Developer Center and open the Web Chat developer tool. For more information, see [Web Chat developer tool quick start](/gettingstarted/developer-tools-web-chats.html "Opens the Web Chat developer tool quick start article").
```

Comment:
This seems to be a broken link

Reply:
Glenn is correct. That feedback made sense for something published on the dev center but not as a readme in github. I am going to cancel this PR.